### DISCUSSION TITLE
Hydra Cloud vs Managed Infrastructure

### DISCUSSION THREAD

**Discussion by hongbo-miao** [hydradatabase/hydra#128] [Q&A] [answered]
Hi Hydra team, at page https://www.hydra.so/pricing, I saw both "Hydra Cloud" and "Managed Infrastructure" have same price.

I am wondering what is the difference between "Hydra Cloud" vs "Managed Infrastructure"?
Is "Managed Infrastructure" more a self-hosted option on for example AWS while we have more controls?

Thank you! 😃

**Comment by wuputah** [comment, marked as answer]
Yes, we have an offering in early access to run our managed Hydra Cloud in your AWS account, so you get all the benefits of Hydra Cloud but it in runs in your account - your EC2 instances, your S3 bucket, your VPC, etc.

If you're interested, please [get it touch](https://www.hydra.so/get-in-touch).
